### ISSUE TITLE
Standardise CSS measurement units to REM

### ISSUE THREAD

**Issue by JulioCesarDias** [conveyordata/data-product-portal#24]
## Summary
There is currently a mix between PX and REM measurement units being used in the CSS files. All the css measurement units should be standardise to REM to improve responsiveness, scalability and accessibility.

### Motivation
Why do we need this improvement? What are the benefits?
This improvement will provide a more consistent approach to sizing and spacing across the UI. REM units are relative and allow for a consistent sizing and spacing across devices with different screen sizes. That will allow users to have a better experience regardless of their device's screen size.
